fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordMetaData.java

```diff
@@ -671,9 +673,14 @@ public static Map<String, Descriptors.FieldDescriptor> getFieldDescriptorMapFrom
 
     @Nonnull
     private static Map<String, Descriptors.FieldDescriptor> getFieldDescriptorMap(@Nonnull final Stream<RecordType> recordTypeStream) {
+        // we should get rid of this method as it is attempting to create an "uber-record" in a brittle way
```

Comment:
use `TODO` to make it stand out more; also please log an issue and link it here

Reply:
done.